test/components/toggle.spec.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import {expect, ClientRenderer, sinon, simulate, waitFor} from 'test-drive-react';
+
+import {Toogle} from '../../src';
+
+describe('<Toggle/>', () => {
+    const clientRenderer = new ClientRenderer();
+    afterEach(() => clientRenderer.cleanup());
+
+    describe('render without options', () => {
+        let onChange: any
+        let renderer: any
+        beforeEach(() => {
+            onChange = sinon.spy();
+            renderer = clientRenderer.render(<Toogle onChange={onChange}/>)
+        })
+        it('should have input', () => {
+            expect(renderer.select('TOGGLE_INPUT')).to.not.null
+        })
+        it('should have default unchecked icon', () => {
+            expect(renderer.select('TOGGLE_UNCHECKED_ICON')).to.not.null
+        })
+        it('should not have default checked icon', () => {
+            expect(renderer.select('TOGGLE_CHECKED_ICON')).to.be.null
+        })
```

Comment:
that is everything is not a null will be positive, but you're expecting DON node there.